src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/CoseSign1Message.cs

```diff
@@ -55,7 +55,7 @@ public static byte[] Sign(byte[] content, RSA key, HashAlgorithmName hashAlgorit
         }
 
         [UnsupportedOSPlatform("browser")]
-        public static byte[] Sign(byte[] content, CoseHeaderMap protectedHeaders, CoseHeaderMap unprotectedHeaders, AsymmetricAlgorithm key, bool isDetached = false)
+        public static byte[] Sign(byte[] content, CoseHeaderMap protectedHeaders, CoseHeaderMap unprotectedHeaders, AsymmetricAlgorithm key, HashAlgorithmName hashAlgorithm, bool isDetached = false)
```

Comment:
I feel like when we chatted we thought that the CoseHeaderMap parameters should be allowed to be null?  Is that just being deferred to a followup? (Or am I misremembering?)

Reply:
> Is that just being deferred to a followup? 

I would prefer to defer that to a subsequent PR. It won't be a breaking change anyway.